modules/layers/src/column-layer/column-layer.js

```diff
@@ -148,24 +180,47 @@ export default class ColumnLayer extends Layer {
         i++;
       }
     }
-    model.setProps({geometry});
+
+    this.state.models.forEach(model => {
+      if (model) {
+        model.setProps({geometry});
+      }
+    });
   }
 
   draw({uniforms}) {
-    const {elevationScale, extruded, offset, coverage, radius, angle} = this.props;
-
-    this.state.model
-      .setUniforms(
-        Object.assign({}, uniforms, {
-          radius,
-          angle: (angle / 180) * Math.PI,
-          offset,
-          extruded,
-          coverage,
-          elevationScale
-        })
-      )
-      .draw();
+    const {
+      elevationScale,
+      wireframe,
+      filled,
+      extruded,
+      offset,
+      coverage,
+      radius,
+      angle
+    } = this.props;
+    const {filledModel, strokeModel} = this.state;
+    const renderUniforms = Object.assign({}, uniforms, {
+      radius,
+      angle: (angle / 180) * Math.PI,
+      offset,
+      extruded,
+      coverage,
+      elevationScale
+    });
+    const numInstances = this.getNumInstances();
+    if (strokeModel && wireframe) {
+      strokeModel.setInstanceCount(numInstances);
+      strokeModel.setUniforms(renderUniforms);
+      strokeModel.setDrawMode(GL.LINES);
```

Comment:
If drawMode and `isWireframe` are always constants, you can pass them during Model construction and avoid setting them for every draw, same for `filledModel`.

Reply:
@kaushikb11 

`drawMode` and `isWireframe` could be set in `_getModels`